python/tvm/contrib/hexagon/build.py

```diff
@@ -149,6 +149,18 @@ def stop_server(self):
         """Stop the RPC server"""
         ...
 
+    def _acquire_device_resources(self):
+        with self.start_session() as session:
+            """Call into device to initialize objects to acquire resources"""
```

Comment:
Linting help: If applicable to the entire function, this string should be moved one line higher, so that it is the docstring of the function.  If applicable only to this `with` scope, it should be a comment instead of a string literal.